pkg/cable/ipsec/strongswan.go

```diff
@@ -36,6 +38,12 @@ const (
 	charonViciSocket = "/var/run/charon.vici"
 )
 
+func init() {
+	cable.AddDriver("", NewStrongSwan) //default
```

Comment:
Looking at this some more, I'm thinking maybe we shouldn't specify the default driver in this manner as the engine's `Endpoint` object would also advertise an empty `Backend` driver field. That doesn't really matter now other than one inspecting the `Endpoint` object from the API server would need knowledge of the internal code to know which driver is being used. But perhaps remote clusters may utilize the `Backend` field in the future. 

We could add a specific `cable.SetDefautDriver("ipsec/strongswan")` call and in, **main.go**:

```
if submSpec.CableDriver == "" {
    submSpec.CableDriver = cable.GetDefautDriver()
}
```

Reply:
**Question**: should we replace `ipsec` to `ipsec/strongswan` ? or assume that if a user specified cable only it should be used as is.

Reply:
yeah - good question - I was wondering that too. It seems to me the `Backend` that's advertised to other clusters should be the actual driver being used, ie if _"ipsec"_ was specified as the `CableDriver` then `Backend` should contain the full name _"ipsec/strongswan"_. This makes me wonder if we're unnecessarily complicating it too much by formally defining a _"protocol[/driver]"_ convention and allowing just _"ipsec"_.  I think we should just keep it simple and take the `CableDriver` as is - for the Strongswan driver we can still use _"ipsec/strongswan"_ as the name just for additional information clarity or just _"strongswan"_.

Reply:
Done